README.rst

```diff
@@ -132,6 +134,8 @@ The above configuration is the minimal required setup. The full set of options s
       'ignored_exceptions': [],
       'transmit_global_variables': True,
       'transmit_local_variables': True,
+      'enforce_payload_size_limit': True, 
+      'log_payload_size_limits': True,
```

Comment:
Maybe we make this `log_payload_size_limit_breaches` or something to be more explicit.